data.json

```diff
@@ -1354,5 +1354,11 @@
     "githubUsername": "hariketsheth",
     "imageUrl": "https://user-images.githubusercontent.com/72455881/134767118-d853cd39-ddc1-4df6-bda1-1a4315f9c855.png",
     "issueNumber": 728
+  },
+  {
+    "name": "Aniket Kumar",
+    "githubUsername": "aniket-24",
+    "imageUrl": "https://user-images.githubusercontent.com/76562459/134800047-8d5db274-3916-486f-9652-ea1e867cbd9d.png",
```

Comment:
I would recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/134812399-6a005576-3b24-46a6-b9bc-7798f79e5bf1.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134812399-6a005576-3b24-46a6-b9bc-7798f79e5bf1.png",
```